client/src/components/landing.tsx

```diff
@@ -2,59 +2,119 @@
+import { Grid } from "@mui/material";
 import { makeStyles } from "@mui/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { Branch } from "@/interfaces/branch";
 
 const useStyles = makeStyles((theme) => ({
   container: {
-    display: "flex",
-    flexDirection: "column",
-    justifyContent: "center",
-    width: "100vw",
-    height: "100vh",
-    padding: "0 15% 0 50vw",
-  },
-  title: {
+    padding: "2.5rem 4rem",
+  },
+  logoContainer: {
+    marginBottom: "3rem",
+    maxWidth: "90vw",
+    width: "18rem",
+    height: "2rem",
+    position: "relative",
+  },
+  h2: {
     fontSize: "3rem",
     fontWeight: "600",
-    marginBottom: "50px",
+    margin: "0",
+  },
+  h3: {
+    fontSize: "1.5rem",
+    fontWeight: "600",
+    margin: ".5rem 0 0 0",
+  },
+  h3Container: {
+    marginTop: "3rem",
+    marginBottom: "1rem",
   },
-  subtitle: {
-    fontSize: "1.8rem",
-    fontWeight: "400",
-    marginBottom: "50px",
+  branchContainer: {
+    position: "relative",
+    height: "100%",
+    width: "fit-content",
   },
-  bandName: {
-    width: "120px",
+  branchName: {
+    fontSize: "1.5rem",
+    fontWeight: "600",
+    marginTop: "1rem",
     textAlign: "center",
     color: theme.palette.text.primary,
   },
   image: {
-    height: 120,
-    width: 120,
+    borderRadius: 9999,
+    objectFit: "cover",
+  },
+  imageContainer: {
+    height: "20vw",
+    width: "20vw",
+  },
+  imageContainerSmall: {
+    height: "30vw",
+    width: "30vw",
   },
 }));
 
-export default function Landing() {
+interface Props {
+  branches: Branch[];
+}
+
+export default function Landing({ branches }: Props) {
   const classes = useStyles();
+  const matches = useMediaQuery("(max-width: 899px)");
 
   return (
     <div className={classes.container}>
-      <h1 className={classes.title}>Drum Roulette</h1>
-      <h2 className={classes.subtitle}>
-        randomly generate band percussion assignments with one click
-      </h2>
-      <h3 className={classes.subtitle}>Please Select Your Branch</h3>
-      <Link href="/selection">
+      <div className={classes.logoContainer}>
         <Image
+          alt="Drum Roulette Logo"
+          src={drLogo}
+          fill
           priority
-          src={navyLogo}
-          alt="Click to select Navy"
-          className={classes.image}
+          style={{ objectFit: "contain" }}
         />
-        <Typography className={classes.bandName}>US Navy Band</Typography>
-      </Link>
+      </div>
+      <h2 className={classes.h2}>Hello!</h2>
+      <h3 className={classes.h3}>
+        Randomly generate band percussion assignments with just a few clicks.
+      </h3>
+      <div className={classes.h3Container}>
+        <h3 className={classes.h3}>Select Branch</h3>
+      </div>
+      <Grid container spacing={{ xs: 2, md: 3 }}>
+        {branches.map((branch) => (
+          <Grid key={branch.branch_id} item xs={6} md={3}>
```

Comment:
could we possibly pull this out into its own component?

Reply:
created a `branchesGrid` component. I could also pull each branch logo/link/caption into its own `branchLogo` component or similar - lmk!